### DISCUSSION TITLE
Any plan to upgrade to `e2sm-kpm-v5.00` and `e2sm-rc-v6.00`?

### DISCUSSION THREAD

**Discussion by OniReimu** [srsran/oran-sc-ric#19] [Ideas]
Hi, I realized new specs are enabled according to here https://specifications.o-ran.org/specifications, any plans?
From my understanding, us directly replacing the asn files won't work. This could be up to if the e2 nodes (from the 5gc) we are connecting to support these new specs or not? Correct me if I am wrong, thanks.

**Comment by pgawlowicz** [comment]
sure, at some point we will update it. Do you have any specific needs that can be addressed only with the newest version?
I have checked and there is little update in the KPM module (mostly related to the REPORT Service Style 255, which allows initiating multiple report action jobs with a single request).

**Reply by OniReimu** [reply]
Thanks for the clarification. Just want to double check if merely replacing the asn files are not enough to make it work?

**Reply by pgawlowicz** [reply]
since the changes are small, it might still work on both sides (RIC and srsRAN E2 agent). But the features like REPORT Service Style 255 are not supported in the e2 agent.
